### PR TITLE
Fix atIndex() to work for swipes and scrolls

### LINKED ISSUE #1461: cannot use Swipe along with atIndex
**Describe the bug**
You get thrown a multiple match error even when using atIndex with a swipe only on iOS. On Android it works.

**To Reproduce**

- [x] I have tested this issue on the latest Detox release and it still reproduces

The line is
```
await element(by.id('background'))
    .atIndex(0)
    .swipe('down', 'fast', 0.7);
```

**Environment (please complete the following information):**
Detox: 12.11.0
React Native: 0.59.8
Node: 10.15.3
Device: iPhone 7
Xcode: 10.2.1
iOS: 12.2
macOS: 10.14.3

**Device and Verbose Detox Logs**
[error.txt](https://github.com/wix/Detox/files/3303579/error.txt)

### PR DESCRIPTION
**Pull request by LeoNatan**
Closes #1461

### CODE REVIEW

**Comment by noomorph** [comment]
Let's add e2e maybe?

**Comment by LeoNatan** [comment]
Let's let it pass first. I have a feeling that the same problem exists in Android. @d4vidi can you pls look at the changes and check if Android has the same issue?

**Comment by LeoNatan** [comment]
```
23:24:05 Jest: "global" coverage threshold for statements (100%) not met: 99.93%
23:24:05 Jest: "global" coverage threshold for branches (100%) not met: 99.79%
23:24:05 Jest: "global" coverage threshold for lines (100%) not met: 99.93%
```

What is this garbage? 🙄

File: detox/test/e2e/02.matchers.test.js [hunk 1]
```diff
@@ -19,6 +19,11 @@ describe('Matchers', () => {
     await element(by.text('Index')).atIndex(index).tap();
     await expect(element(by.text('First button pressed!!!'))).toBeVisible();
   });
+  
+  it('should be able to swipe elements matched by index', async () => {
+    const index = device.getPlatform() === 'ios' ? 2 : 0;
```
Comment (noomorph) [inline detox/test/e2e/02.matchers.test.js]: @LeoNatan, why 2 for iOS? Is it done on purpose, so that tests don't fail on Android?

Comment (LeoNatan) [reply detox/test/e2e/02.matchers.test.js]: Yes, the order is different between iOS and Android. That's why the `atIndex()` API is terrible. There is no control over order.

**Pull request merged**